skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -83,6 +82,19 @@ actual open class SkiaLayer internal constructor(
             override fun getInputMethodRequests(): InputMethodRequests? {
                 return this@SkiaLayer.inputMethodRequests
             }
+
+            override fun requestFocus(cause: FocusEvent.Cause?) {
+                if (canReceiveFocus(cause)) {
+                    super.requestFocus(cause)
+                }
+            }
+
+            override fun requestFocusInWindow(cause: FocusEvent.Cause?): Boolean {
+                return canReceiveFocus(cause) && super.requestFocusInWindow(cause)
+            }
+
+            private fun canReceiveFocus(cause: FocusEvent.Cause?) = cause != FocusEvent.Cause.MOUSE_EVENT ||
```

Comment:
Why only check to MOUSE_EVENT?
How about "Tab" key on keyboard?

Reply:
By `setRequestFocusEnabled` description, it should only affect mouse:
```
Provides a hint as to whether or not this JComponent should get focus. This is only a hint, and it is up to consumers that are requesting focus to honor this property. This is typically honored for mouse operations, but not keyboard operations
```
It is used in Compose to prevent changing focus on ComposePanel clicks, but keep focus changing when we press Tab